src/main/java/heylo/tasks/Deadline.java

```diff
@@ -0,0 +1,38 @@
+package heylo.tasks;
+
+import heylo.util.DateFormatter;
+
+import java.time.LocalDate;
+
+/**
+ * Represents a task with a deadline.
+ */
+public class Deadline extends Task {
+
+    protected LocalDate deadline;
+
+    /**
+     * Create a task with a deadline.
+     *
+     * @param description String description of the task.
+     * @param deadline    String date of the deadline.
```

Comment:
There's some unnecessary spacing between "deadline" and "String date of the deadline." Same comment for Event constructor.